scripts/zkai_attention_kv_proof_route_selector_gate.py

```
@@ -333,13 +376,30 @@ def _load_stwo_native_masked_sequence_module():
     return module
 
 
+def _load_quantized_softmax_receipt_module():
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_load_quantized_softmax_receipt_module`

(ANN202)